.github/workflows/mark-issue-stale.yml

```diff
@@ -9,19 +9,23 @@ jobs:
   stale:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/stale@v3
+      - uses: actions/stale@v8
         with:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
+          # Does not perform action on issue/PR.
+          debug-only: true
           # Message to be added to stale issues.
-          stale-issue-message: 'This issue is stale because it has been 180 days with no activity. You can keep the issue open by adding a comment. If you do, please provide additional context and explain why you’d like it to remain open. You can also close the issue yourself — if you do, please add a brief explanation and apply one of relevant issue close labels.'
+          stale-issue-message: 'This issue is stale because 720 days have passed with no activity. A member of @kitkat-team will review and and if required close this issue within 7 days. If you disagree and would like this issue to remain open, please provide additional context, updated reproduction steps and/or screenshots.'
```

Comment:
**chore:** remove the duplicate 'and' -> "A member of @kitkat-team will review **and** and if required"